internal/graph/dotgraph.go

```diff
@@ -450,14 +449,9 @@ func tagDistance(t, u *Tag) float64 {
 }
 
 func (b *builder) tagGroupLabel(g []*Tag) (label string, flat, cum int64) {
-	formatTag := b.config.FormatTag
-	if formatTag == nil {
-		formatTag = measurement.Label
-	}
-
 	if len(g) == 1 {
 		t := g[0]
-		return formatTag(t.Value, t.Unit), t.FlatValue(), t.CumValue()
```

Comment:
I can see that formatTag() is invoked even for a single tag (when len(g) == 1). So I am not entirely clear why the bug only manifested for grouped tags, could you elaborate?